sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -2,13 +2,8 @@
 
 ## 5.2.0 (Unreleased)
```

Comment:
make sure you have a PR to update the date

Reply:
Will do in another PR 👍